security/pkg/nodeagent/sds/server.go

```diff
@@ -29,8 +32,13 @@ import (
 	"istio.io/pkg/version"
 )
 
-const maxStreams = 100000
-const maxRetryTimes = 5
+const (
+	// base HTTP route for debug endpoints
+	debugBase     = "/debug"
+	debugPort     = 8080
```

Comment:
Can we reuse `ctrlz` port? Rather than having to redefine a new hardcoded port? so that the path will be 

- ":8080/ctrlz"
- ":8080/debug/sds/workload"
- ":8080/debug/sds/gateway"

Reply:
fwiw pilot adds debug under 8080 and controlz under 9876 - doesn't mean its the best way to do it though

Reply:
seems nodeagent ctrlz  DefaultOptions which set port to 9876 ? (https://github.com/istio/istio/blob/master/vendor/istio.io/pkg/ctrlz/options.go#L33) 

instead of hardcode the port number, could we use a env var like [MonitoringPort](https://github.com/istio/istio/blob/master/security/cmd/node_agent_k8s/main.go#L123) ?